### DISCUSSION TITLE
How to see if weak consistency is enabled

### DISCUSSION THREAD

**Discussion by basvdberg** [yonileibowitz/yonileibowitz.github.io#14] [Q&A]
Hi, 
I've enabled weak consistency like this:
 .alter-merge workload_group default ``````
{
  `"QueryConsistencyPolicy`": {
     `"QueryConsistency`": {
    `"IsRelaxable`": true,
    `"Value`": `"Weak`"
  },
  `"CachedResultsMaxAge`": {
    `"IsRelaxable`": true,
    `"Value`": `"08:00:00`"
  }
}
} ``````

I wanted to ask how to check if this setting is applied correctly, because this command :
.show cluster policy query_weak_consistency 
is not showing it. 

But while typing this question I realised that I had to type:
.show workload_groups 

this will show:
 },
  "QueryConsistencyPolicy": {
    "QueryConsistency": {
      "IsRelaxable": true,
      "Value": "Weak"
    },
    "CachedResultsMaxAge": {
      "IsRelaxable": true,
      "Value": "08:00:00"
    }

**Comment by y0nil** [comment]
1. the appropriate place to post technical questions regarding kusto is StackOverflow.
2. the `.show` command you're using shows you a _different_ policy - [the cluster-level query weak consistency policy](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/management/query-weak-consistency-policy), and not the one you've set - [the workload-group-level query consistency policy](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/management/query-consistency-policy).
3. it sounds like you already found your answer.
